packages/pg/package.json

```diff
@@ -24,9 +24,10 @@
     "packet-reader": "1.0.0",
     "pg-connection-string": "^2.5.0",
     "pg-pool": "^3.4.1",
-    "pg-protocol": "^1.5.0",
```

Comment:
How come `pg-protocol` is being removed?  Don't you need this for the new algorithm that has been added?